apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/styles/starter-page-templates-editor.scss

```diff
@@ -146,10 +146,12 @@ body.admin-bar:not( .is-fullscreen-mode ) .page-template-modal-screen-overlay {
 		box-sizing: content-box;
 		position: relative;
 		pointer-events: none;
-		opacity: 1;
 
-		&.is-rendering {
-			opacity: 0.5;
+		.components-spinner {
+			position: absolute;
+			top: 50%;
+			left: 50%;
+			margin: 0 0 0 -9px;
```

Comment:
What's the `-9px` magic number?

Reply:
:magic: :-D the spinner width is 18px, so it moves the spinner at left with the half this value.

Reply:
> :magic: :-D

😄 Sorry. Not sure if everyone is aware of this as a "thing" but Harry Roberts has this down as one of his things to look out for in CSS codebase reviews

https://csswizardry.com/2012/11/code-smells-in-css/#magic-numbers

> ...the spinner width is 18px

A small point but I have found it is helping to have comments explaining why these numbers exist or use well named Sass variables to make the code self-documenting. What do you think? Is it overkill?

```
margin: 0 0 0 -#{$spinner-width / 2};
```